build.xml

```diff
@@ -29,6 +29,12 @@
         </exec>
     </target>
 
+    <target name="create-index" depends="prepare" description="Create index">
```

Comment:
nechápu - migrace db na connection se taky pouštěj ručně, tohle souvisí s updatem na vyšší syrup, v unit testech to snad bejt nemusí, ne?